chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ApiKeyController.java

```diff
@@ -103,7 +104,7 @@ private String delete(final String key, final String userApiKey, final String us
     }
 
     @ApiOperation(value = "List all API keys that have been created.",
-            notes = "This service is only available to CHPL users with ROLE_ADMIN or ROLE_ONC.")
+            notes = "Security Restrictions: ROLE_ADMIN, ROLE_ONC, or ROLE_ONC_STAFF")
```

Comment:
Should probably Ctrl-F this value. I'm going to stop commenting

Reply:
Addressed - plus all others...